docs/migration/v11.md

```diff
@@ -540,6 +540,18 @@ can be shortened to `scss/components/<component>` since Dart Sass resolves
 @use '@carbon/react/scss/components/accordion';
 ```
 
+#### Component tokens
+
+To use component tokens, you'll need to `@use` them:
+
+```
+@use '@carbon/react/scss/components/button/tokens';
+
+.my-selector {
+  background: tokens.$button-primary;
```

Comment:
Would it be simpler to use `scss/components/button` and `button.$button-primary` or do you think it's more helpful to use the token entrypoint explicitly?

Reply:
Good idea, I'll update